solutions/kubernetes/install-perfline.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash -x
```

Comment:
Please move scripts to https://github.com/Seagate/cortx-re/tree/main/scripts/performance